### DISCUSSION TITLE
Where can I find the rego package lib.result?

### DISCUSSION THREAD

**Discussion by Talkapon** [aquasecurity/trivy#6815] [Q&A] [answered]
### Question

I fail to run conftest on trivy checks since many of them use result.new. I am getting this error: rego_type_error: undefined function result.new.
I was wondering where is this package, as it is clearly not in the main branch.

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by nikpivkin** [comment, marked as answer]
Hi @Talkapon !

This is a custom function that is [registered](https://github.com/aquasecurity/trivy/blob/main/pkg/iac/rego/custom.go#L15) in Trivy. Read more about the custom functions [here](https://www.openpolicyagent.org/docs/latest/extensions/#custom-built-in-functions-in-go).

**Reply by Talkapon** [reply]
Thanks, in that case you may consider fixing this: https://github.com/aquasecurity/trivy-checks/tree/main/checks/kubernetes#readme

it advises to run conftest as is and that is not registering this custom function

**Reply by nikpivkin** [reply]
I'll create an issue for that.
